modules/nf-scil/reconst/frf/meta.yml

```diff
@@ -42,6 +45,27 @@ input:
       description: Nifti b0 binary mask.
       pattern: "*.{nii,nii.gz}"
 
+  - wm_mask:
+      type: file
+      description: Nifti binary WM mask.
+      pattern: "*.{nii,nii.gz}"
+
+  - gm_mask:
+      type: file
+      description: Nifti binary GM mask.
+      pattern: "*.{nii,nii.gz}"
+
+  - csf_mask:
+      type: file
+      description: Nifti binary CSF mask.
+      pattern: "*.{nii,nii.gz}"
+
+  - method:
+      type: string
+      description: Reconstruction method. Either ssst_frf or msmt_frf. This choice
+        change the input masks and the output FRF, be careful.
+      pattern: "*_frf"
```

Comment:
I don't see this in the module input. Since it is provided using the `task.ext.set_method`, you can remove this.